homeassistant/components/hassio/addon_manager.py

```diff
@@ -113,6 +111,14 @@ def __init__(
         self._restart_task: asyncio.Task | None = None
         self._start_task: asyncio.Task | None = None
         self._update_task: asyncio.Task | None = None
+        self._supervisor_client: SupervisorClient | None = None
+
+    @property
+    def supervisor_client(self) -> SupervisorClient:
```

Comment:
Why do we need to expose this attribute outside the instance?

Reply:
Fair point. Don't think we need to. I just wanted to make it a property to actually get and set the internal attribute outside of the constructor, but it can be private within the class